2-virtual-machines-containers/2.2-local-development.md

```diff
@@ -7,6 +7,19 @@ So far the tools we have looked at for building and running images are great for
 ![](img2/developer_light.svg ':size=150x150 :class=light-mode-icon')
 ![](img2/developer_dark.svg ':size=150x150 :class=dark-mode-icon')
 
+?> Once again if you have an M1 Macbook, default vagrant will not work. As Vagrant by default uses virtualbox as its provider & virtualbox does not yet work on M1 macs (as of 23/02/2023) this presents a problem.
```

Comment:
This is such a shame for this section of the bootcamp. But after trying to fight it and realizing there is not an easy solution I think this is a great alternative exercise. I hope the VirtualBox team will have a working build for ARM processors soon 🤞